api_tests/nodes/views/test_node_list.py

```diff
@@ -2245,6 +2253,15 @@ def test_bulk_delete_invalid_payload_one_not_found(self, app, user_one, public_p
         res = app.get(public_project_one_url, auth=user_one.auth)
         assert res.status_code == 200
 
+    def test_bulk_delete_project_with_component(self, app, user_one, public_project_parent, public_component, url):
+        new_payload = {'data': [{'id': public_project_parent._id, 'type': 'nodes'}, {'id': public_component._id, 'type': 'nodes'}]}
+        res = app.delete_json_api(url, new_payload, auth=user_one.auth, expect_errors=True, bulk=True)
+        assert res.status_code == 400
+
+        new_payload = {'data': [{'id': public_component._id, 'type': 'nodes'}, {'id': public_project_parent._id, 'type': 'nodes'}]}
+        res = app.delete_json_api(url, new_payload, auth=user_one.auth, expect_errors=True, bulk=True)
```

Comment:
This looks like it should not expect errors, true?

Reply:
In case I worded that poorly: because this request expects a success, `expect_errors` should probably be set to `False`.

Reply:
Ah, right. Will change that.